README.md

```diff
@@ -24,7 +26,7 @@ Each version of the specification is available under a v&lt;#MAJOR&gt;.&lt;#MINO
 * [APIs/RegistrationAPI.raml](APIs/RegistrationAPI.raml) -- Normative specification of the NMOS Registration API
 * [APIs/QueryAPI.raml](APIs/QueryAPI.raml) -- Normative specification of the NMOS Query API
 * [APIs/schemas/](APIs/schemas/) -- JSON schemas used by API specifications
-* [APIs/generateHTML](APIs/generateHTML) -- Tool to create HTML browsable version of the API specifications. Requires raml2html (https://github.com/kevinrenskers/raml2html).
+* [APIs/generateHTML](APIs/generateHTML) -- Tool to create HTML browsable version of the API specifications. Requires raml2html <https://github.com/kevinrenskers/raml2html>.
```

Comment:
Similarly, here, actually!

```suggestion
* [APIs/generateHTML](APIs/generateHTML) -- Tool to create HTML browsable version of the API specifications. Requires [raml2html](https://github.com/kevinrenskers/raml2html).
```

Reply:
I agree and will update all the specs accordingly.